WalletWasabi/WalletWasabi.csproj

```diff
@@ -69,12 +51,39 @@
     <None Update="TorDaemons\Data.zip">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+  </ItemGroup>
+
+  <ItemGroup Condition="$(WasabiPackagerRuntimeIdentifier.StartsWith('linux')) Or $(WasabiPackagerRuntimeIdentifier) == ''">
```

Comment:
This will be true `$(WasabiPackagerRuntimeIdentifier) == ''` if someone just type dotnet publish?

Reply:
Yes. 

Also, it will be hit when you debug/run WasabiWallet application in an IDE.

Reply:
It seems that you can improve it using: https://stackoverflow.com/a/48714896

Reply:
I replaced your custom variable with RuntimeIdentifier, check it pls

Reply:
@molnard If it works for you, then great. But it needs to be tested on all platforms. I can test on Windows 10.

Reply:
Basically the packager runs from win (we need wsl that is win only) and osx but only for signing.

Reply:
It would be nice to have Packager available for other platforms too. Even if it may not correctly create deterministic builds but it allows other developers to test any Packager-related PRs.